packages/lab/stories/pill-next/pill-next.stories.tsx

```diff
@@ -0,0 +1,40 @@
+/**
+ * **NOTE**
+ *
+ * This file has same title with the .mdx companion file.
+ * Stories defined in this/that file may not be shown in the navigation tree after hot reloading.
+ * Refresh of a page is needed.
+ */
+
+import { SaltProvider } from "@salt-ds/core";
+import { PillNext } from "@salt-ds/lab";
+import { FavoriteIcon } from "@salt-ds/icons";
+import { ComponentMeta, ComponentStory } from "@storybook/react";
+
+export default {
+  title: "Lab/Pill Next",
+  component: PillNext,
+} as ComponentMeta<typeof PillNext>;
+
+export const Default: ComponentStory<typeof PillNext> = () => {
+  const handleClick = () => {
+    console.log("clicked");
+  };
+  return <PillNext onClick={handleClick}>Clickable Pill</PillNext>;
+};
```

Comment:
I'd say the Default example would be non-clickable and this one should be the Clickable example

Reply:
they're all clickable now, i think non clickable is going to be separate tag component or something